src/dune_lang/pform.mli

```diff
@@ -20,6 +20,7 @@ module Var : sig
     | Ocaml_version
     | Ocaml
     | Ocamlc
+    | Ocamldebug
```

Comment:
This could be an independent change.

Reply:
Yes, I will split this out to go with the "dune debug" PR.